dev/tools/bin/get_files_changed.dart

```diff
@@ -0,0 +1,57 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert';
+import 'dart:io' as io;
+
+import 'package:args/args.dart';
+
+final ArgParser _argParser =
+    ArgParser()
+      ..addOption(
+        'since',
+        help: 'What previous SHA to compare the current git state to.',
+        defaultsTo: 'HEAD^',
+      )
+      ..addOption(
+        'output',
+        help: 'What format to output in.',
+        defaultsTo: io.stdout.hasTerminal ? 'text' : 'json',
+        allowed: <String>['text', 'json'],
+      );
+
+void main(List<String> args) async {
+  final ArgResults argResults = _argParser.parse(args);
+
+  // Get a list of files changed between this commit and the base SHA.
+  final List<String> filesChanged;
+  {
```

Comment:
Why the block?

Reply:
It's not necessary, just encloses some local variables (i.e. `stdout`) that have no purpose outside the block.